java-runtime-builder-app/pom.xml

```diff
@@ -84,6 +84,11 @@
       <artifactId>commons-io</artifactId>
       <version>2.5</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
```

Comment:
It's actually required by Guice - otherwise it will refuse to inject a null reference.